docs/blog/posts/using-vega-protocol-with-hummingbot/index.md

```diff
@@ -0,0 +1,172 @@
+
+
+**Metamask Snap** (recommended)
+
+If you're already a MetaMask user, you won't need to create a Vega Wallet to create and use a Vega key.
+
+Once you have Metamask installed, open either Chrome or Firefox and go to [Vega Console](https://console.vega.xyz/)
+
+Click on either "Getting Started" at the top right or "Connect Wallet" at the bottom right
+
+[![image](image0.png)](image0.png)
+
+You should get a popup on the screen, select the second option "Install Vega Metamask Snap"
+
+[![image](image2.png)](image2.png)
+
+Metamask will open a series of requests pop-ups, review them and then click "Connect", "Install", "Confirm"
+
+[![image](image1.png)](image1.png)
+
+Once this is done, click on "Connect" again or if you still have the previous popup open then select "Connect via Metamask Snap" one more time. You should get a prompt on the screen and click "I Agree" to continue
+
+[![image](image3.png)](image3.png)
+
+Once installed and connected to the site, you will see your `SNAP KEY` in the top right with the option, when clicked, to copy the address, this is your `Vega Party ID` which will be used within the system when configuring Hummingbot.
+
+You're now ready to deposit or transfer assets and use your new Vega keypair.
+
+### Deposit Funds
+
+!!! Note
+    Please make sure you have enough balance in your wallet including ETH for gas fees. 
+
+In order to trade on Vega we'll need to deposit funds first. Click on the Deposit button in the right hand side of the screen
+
+[![image](image5.png)](image5.png)
+
+You may get a message asking you to approve first the use of the chosen asset before you are able to deposit. Make sure to approve this.
+
+[![image](image6.png)](image6.png)
+
+In the Deposit window double check the `Vega Key` then click the Asset drop down box and select the Asset you want to deposit. In the example below we are going to select `USDT Faucet` and enter in `500` under the Amount. Double check the above details again then once satisfied, click Deposit. 
+
+[![image](image7.png)](image7.png)
+
+You'll then need to approve the transaction in Metamask - click "Confirm" to confirm the transaction. 
+
+[![image](image8.png)](image8.png)
+
+Once this is done, you'll just need to wait for the transaction to be confirmed. You should get a confirmation message once this is done. 
+
+[![image](image9.png)](image9.png)
+
+
+## Install Hummingbot
+
+There are two main methods to install Hummingbot:
+
+### Docker
+
+For most new users, we recommend installing Hummingbot using Docker.
+
+- [Docker Quickstart Guide](https://hummingbot.org/academy-content/docker-installation-guide/)
+
+### Source
+
+We recommend installing Hummingbot from source if you meet any of the following criteria:
+
+- You want to customize or extend the Hummingbot codebase.
+- You want to build new components like connectors or strategies.
+- You want to learn how Hummingbot works at a deeper, technical level.
+
+Guides for various environments:
```

Comment:
Add a space before bullet points to prevent this
<img width="512" alt="Screenshot 2024-02-08 at 6 09 42 AM" src="https://github.com/hummingbot/hummingbot-site/assets/1795030/2aae4045-b134-4b0b-a77a-a225d1333503">